src/components/SideDrawer.vue

```diff
@@ -1,246 +1,114 @@
-            </v-icon>
+            <v-icon class="icon-20">close</v-icon>
           </v-btn>
-          <div v-show="updateProfileTip">
-            <v-card-text class="h6 pb-0">
-              <v-row class="mb-1">
-                <p class="body-lg mt-3 text--base-darkest mb-0">
-                  ATAT uses Global Directory for CAC authentication. GD is an
-                  enterprise identity solution that provides a trusted single
-                  source of truth for digital authentication for every DoD
-                  employee.
-                </p>
-                <p class="body-lg mt-2 text--base-darkest mb-0">
-                  When you update your contact information on the
-                  <a href="https://idco.dmdc.osd.mil/idco/" class="link-body-md"
-                    >ID Card Office Online (IDCO) website</a
-                  >, it is automatically updated anywhere you use your CAC,
-                  saving you the time and hassle of updating your information on
-                  multiple DoD websites.
-                </p>
-              </v-row>
-            </v-card-text>
-          </div>
-        </v-col>
-      </v-row>
-    </v-container>
-  </v-navigation-drawer>
+        </div>
+      </div>
+      <ProfileDrawer
+        v-if="sideDrawerType === 'profile'"
+        :drawerWidth="drawerWidth"
+        :drawerHeight="getHeight"
+      ></ProfileDrawer>
+      <SubmitDrawer
+        v-if="sideDrawerType === 'submit'"
+        :drawerWidth="drawerWidth"
+        :drawerHeight="getHeight"
+      ></SubmitDrawer>
+    </v-navigation-drawer>
+  </v-slide-x-reverse-transition>
 </template>
 <script lang="ts">
 import Vue from "vue";
 
-import { Component, Prop } from "vue-property-decorator";
-@Component({})
+import { Component, Prop, Watch } from "vue-property-decorator";
+import ProfileDrawer from "./SideDrawerComponents/ProfileDrawer.vue";
+import SubmitDrawer from "./SideDrawerComponents/SubmitDrawer.vue";
+@Component({
+  components: {
+    ProfileDrawer,
+    SubmitDrawer,
+  },
+})
 export default class SideDrawer extends Vue {
-  private contactInfoTip = false;
-  private updateProfileTip = false;
-
   @Prop({ default: "400" }) private drawerWidth!: string;
 
-  private user = this.$store.getters.getUser;
-  get showScrollbar(): string {
-    const show =
-      window.innerHeight < 850 || this.updateProfileTip || this.contactInfoTip;
-    return show ? "expandedSidebarDiv" : "";
+  get sideDrawerType(): string {
+    return this.$store.state.sideDrawerType;
+  }
+
+  get getTitle(): string {
+    let title = "";
+    switch (this.sideDrawerType) {
+      case "profile":
+        title = "Your Profile";
+        break;
+      case "submit":
+        title = "Learn More";
+        break;
+      default:
+        break;
+    }
+    return title.toUpperCase();
+  }
+
+  get getHeight(): string {
+    const drawerHeight = window.innerHeight;
+    return drawerHeight + "px";
```

Comment:
could just be `return window.innerHeight + "px";` if not using variable elsewhere